### DISCUSSION TITLE
CLARITY BOOST

### DISCUSSION THREAD

**Discussion by DVergil** [redphx/better-xcloud#166] [General]
Why is better xcloud's "clarity boost" so heavy? Even on my IOS it runs very heavily

**Comment by DVergil** [comment]
In kiwi, the decode time increases a lot and begins to have delay and stuttering when I increase more and more

**Comment by redphx** [comment]
Which device are you using?

**Comment by DVergil** [comment]
I tested on an IOS in the latest version and on an android tv version 9

**Comment by DVergil** [comment]
I used the safari browser on IOS and Kiwi on Android TV
